matscipy/calculators/manybody/potentials.py

```diff
@@ -74,3 +78,53 @@ def hessian(self, r_p, xi_p):
             np.zeros_like(xi_p),
             np.zeros_like(xi_p),
         ])
+
+class HarmonicAngle(Manybody.Theta):
+    """
+    Implementation of a harmonic angle interaction.
+    """
+
+    def __init__(self, k0=0.1, theta0=149.47):
+        self.k0 = k0
+        self.theta0 = theta0
+
+    def __call__(self, rij, rik, rjk):
+        rsq_ij = rij**2
+        rsq_ik = rik**2
+        rsq_jk = rjk**2
+
+        theta_c = np.arccos((rsq_ij + rsq_ik - rsq_jk) / (2 * rij * rjk))
```

Comment:
In LAMMPS, if atoms `[A1, A2, A3] ` are defining an angle, then  `A2` is the center atom. We should discuss which of `i, j, k` is the center atom in our formulation. In `explicit_forms/harmonic.py` I set `j` to be the center atom, like in LAMMPS. If we keep that convention, then the cosine should be `(rsq_ij + rsq_jk - rsq_ik) / (2 * rij * rjk)`.